libopenage/datastructure/constexpr_map.h

```diff
@@ -268,15 +238,13 @@ class ConstMap<K, V, 0> {
 	 * Fetching a value from the end element aborts as it has no entry.
 	 */
 	constexpr const inline V &fetch(const K &) const {
-		return this->missing_key();
+		// A key is not in the map.
+		// This function should never be called. If it is called,
+		// you requested a nonexisting key.
+		return *static_cast<const V*>(nullptr);
```

Comment:
This (second answer) might be usable here

https://stackoverflow.com/questions/37440370/is-it-possible-to-check-if-const-value-is-known-at-compile-time